BoxContentSDK/BoxContentSDK/Operations/BOXAPIAuthenticatedOperation.m

```diff
@@ -8,7 +8,6 @@
 
 #import "BOXAPIAuthenticatedOperation.h"
 
-#import "BOXAPIJSONOperation.h"
```

Comment:
THis was a code smell. An abstract base class including the header of a subclass???